docs/modules/deploy/pages/choosing-a-deployment-option.adoc

```diff
@@ -1,7 +1,7 @@
 = Choosing an Application Topology
 :description: Hazelcast offers two topologies to use Hazelcast: embedded in a Java application or in a client/server topology. To help you choose a topology that meets your requirements, this topic describes each option.
 :page-aliases: ROOT:topologies.adoc
-:url-cloud-signup: https://cloud.hazelcast.com/sign-in?next=/
+:url-cloud-signup: https://cloud.hazelcast.com/sign-in?next=/?utm_source=docs-website
```

Comment:
```suggestion
:url-cloud-signup: https://cloud.hazelcast.com/sign-in?utm_source=docs-website
```

Reply:
Not sure what the next=/ was meant to do but I did test my first version and it worked so I left it in, but you're right we should prob correct this at same time too